### DISCUSSION TITLE
Manually Setting Verification Token, and Using the Verification URL

### DISCUSSION THREAD

**Discussion by AbdelrahmanSadat** [nextauthjs/next-auth#10361] [Help]
I'm trying to setup and api endpoint where another server will authenticate the user, and then send me a request to create a record for them on my side, and then I can respond with the verification url.

I already create a verification token in my db, and then send the verification url. But when the user tries to access it on their side, they get the "unable to sign in" page, without further information.

So basically what I want to know is: if I create a verificationToken manually in the db, and then send it with the next-auth callback url, shouldn't it log in the user? Can I work around that "unable to sign in" page, or get more info on what's causing it?
